src/views/List.jsx

```diff
@@ -20,52 +20,59 @@ export function List({ data, listPath }) {
 	});
 
 	function handleInputChange(e) {
+		e.preventDefault();
 		setInput(e.target.value);
 	}
 
 	function clearSearch() {
 		setInput('');
 	}
 
+	const listName = listPath?.substring(listPath.indexOf('/') + 1);
+
+	if (data.data.length === 0) {
+		return (
+			<>
+				<h2>{listName}</h2>
+				<AddItemForm listPath={listPath} />
+			</>
+		);
+	}
+
 	if (data.loading) {
 		// If data is not loaded yet, render a loading indicator
 		return <p>Loading...</p>;
 	}
 
 	return (
 		<>
-			<form action="">
-				<label htmlFor="searchItems">Search Items: </label>
-				<input
-					onChange={handleInputChange}
-					id="searchItems"
-					value={input}
-					type="text"
-				/>
-			</form>
-			<button onClick={clearSearch}>X</button>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<h2>{listName}</h2>
+			<div className="searchInput">
+				<form action="" onSubmit={(e) => e.preventDefault()}>
+					<label htmlFor="searchItems">Search your shopping list: </label>
+					<input
+						onChange={handleInputChange}
+						id="searchItems"
+						value={input}
+						type="text"
+					/>
+				</form>
+				<button onClick={clearSearch}>Clear</button>
+			</div>
 
-			<ul>
-				{data.length === 0 && (
-					<div>
-						<p>There are no items in this list!</p>
-						<button onClick={() => navigate('/manage-list')}>
-							Add item to list
-						</button>
-					</div>
-				)}
-			</ul>
-			{data.length > 0 && filteredItems.length === 0 && (
+			{data.data.length > 0 && filteredItems.length === 0 && (
 				<div>
 					<p>No match found for that filter query.</p>
 				</div>
 			)}
+			<AddItemForm listPath={listPath} />
+			<br />
+			<br />
 			{filteredItems.map((item) => (
 				<ListItem key={item.id} item={item} listPath={listPath} />
 			))}
+			<br />
```

Comment:
I would avoid user `<br/>` to achieve spacing would suggest using a div to apply margin or padding to create space.